README.md

```diff
@@ -40,9 +40,10 @@ The app generator installs and configures the [grunt tasks](#running-the-grunt-t
 |Prompt|Description|Default|
 |------|-----------|-------|
 |Type of widget(s) to be generated|Whether you want to build 2D or 3D widgets|2D|
-|Web AppBuilder install root|The root folder where you installed (unzipped) the Web AppBuilder Developer Edition|[USER_HOME_FOLDER]/arcgis-web-appbuilder-1.3|
+|Web AppBuilder install root|The root folder where you installed (unzipped) the Web AppBuilder Developer Edition|[USER_HOME_FOLDER]/WebAppBuilderForArcGIS|
 |Web AppBuilder application|The name of the application you would like the grunt task to sync your code with|None|
 |Would you like to use SASS for CSS preprocessing?|If you choose yes, you can utilize features from [SASS](http://sass-lang.com/) like nesting, variables, etc.|Yes|
+|Would you like to use Livereload to auto-refresh your browser? ([Help](#livereload))|Yes|
```

Comment:
That's a good point. The only problem I can think of is if our server is on the livereload port (35729) and they don't realize it, and they try to start up some other service that tries to access that same livereload port and the user is confused that it's already in use because they didn't realize we're serving on that port already. I think this risk can be mitigated via a note in the project documentation though, so now that I think about it, I'm ok with just defaulting to livereload. I'll make that change unless there are any other objections.

Reply:
The livereload port can be configured, so we could put something in the documentation on changing the port: https://github.com/gruntjs/grunt-contrib-watch#optionslivereload